example.env

```diff
@@ -33,6 +33,9 @@ FACEBOOK_SECRET=secret_for_facebook_oauth
 GOOGLE_OAUTH2_CLIENT_ID=client_id_for_google_oauth
 GOOGLE_OAUTH2_CLIENT_SECRET=client_secret_for_google_oauth
 
+# Default sender email address
```

Comment:
There's no need for a comment that says exactly the same thing as the constant name. 😅